lib/storages/sequelize.js

```diff
@@ -41,7 +43,9 @@ module.exports = redefine.Class({
       // note 'sequelize' or 'model' is required
       modelName: 'SequelizeMeta',
       // note 'tableName' (optional) also supported
-      columnName: 'name'
+      columnName: 'name',
+      // You may need to override this for utf8mb4 charsets
+      columnLength: 255
```

Comment:
So instead of `columnLength` we could use `columnType` which can then either be `Sequelize.STRING(something)` or just a plain string containg the respective information. Opinion?